inference-engine/src/mkldnn_plugin/nodes/gather.cpp

```diff
@@ -22,18 +22,41 @@ namespace InferenceEngine {
 namespace Extensions {
 namespace Cpu {
 
+using MKLDNNPlugin::TensorDescCreatorTypes;
+
 class GatherImpl: public ExtLayerBase {
 public:
+    static bool isSupportedParams(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
```

Comment:
just suggestion
maybe ```isPluginSupported```?
because we validate not only parameters but also node version

Reply:
Renamed to `isSupportedOperation()`